core/frontend/package.json

```diff
@@ -3,6 +3,46 @@
   "version": "0.1.0",
   "private": true,
   "vue": {
+    "devServer": {
+      "proxy": {
+        "^/docker": {
+          "target": "http://companion.local/"
+        },
+        "^/version-chooser": {
+          "target": "http://companion.local/"
+        },
+        "^/mavlink-camera-manager": {
+          "target": "http://companion.local/"
+        },
+        "^/system-information": {
+          "target": "http://companion.local/"
+        },
+        "^/mavlink2rest": {
+          "target": "http://companion.local/"
+        },
+        "^/ardupilot-manager": {
+          "target": "http://companion.local/"
+        },
+        "^/terminal": {
+          "target": "http://companion.local/"
+        },
+        "^/file-browser": {
+          "target": "http://companion.local/"
+        },
+        "^/wifi-manager": {
+          "target": "http://companion.local/"
+        },
+        "^/cable-guy": {
+          "target": "http://companion.local/"
+        },
+        "^/helper": {
+          "target": "http://companion.local/"
+        },
+        "^/logviewer": {
```

Comment:
Can we make this list alphabetic ?

Reply:
This is following the list in nginx.conf. we should probably make both

Reply:
Agree.